content/docs/howto/java.md

```diff
@@ -384,6 +384,78 @@ docker run --rm --tty \
+
+To enable, set `BP_LIVE_RELOAD_ENABLED=true`. If `BP_LIVE_RELOAD_ENABLED` is true at build-time then the [Executable Jar Buildpack][bp/executable-jar] or the [Dist Zip Buildpack][bp/dist-zip] will request that [`watchexec`](https://github.com/watchexec/watchexec/) is installed and they will contribute a default process type named `reload`. This process type will use `watchexec` as the primary process and Java as a child process of `watchexec`.
+
+**Example**: Reloadable Process
+
+The following commands build an image with a default reloadable process type.
+
+{{< code/copyable >}}
+pack build samples/java \
+  --path java/jar \
+  --env BP_LIVE_RELOAD_ENABLED=true
+{{< /code/copyable >}}
+
+The output will confirm a `reload` process type has been added.
+
+{{< code/copyable >}}
+Paketo Executable JAR Buildpack v5.4.0
+  https://github.com/paketo-buildpacks/executable-jar
+  Class Path: Contributing to layer
+    Writing env/CLASSPATH.delim
+    Writing env/CLASSPATH.prepend
+  Process types:
+    executable-jar: java org.springframework.boot.loader.JarLauncher (direct)
+    reload:         watchexec -r java org.springframework.boot.loader.JarLauncher (direct)
+    task:           java org.springframework.boot.loader.JarLauncher (direct)
+    web:            java org.springframework.boot.loader.JarLauncher (direct)
+{{< /code/copyable >}}
+
+You may also run `pack inspect samples/java` which will show you the process types.
+
+### Using Tilt with Paketo Buildpacks
+
+You can use [Tilt](https://tilt.dev/) with the Paketo Java Buildpacks. It requires a custom build command in your `Tiltfile`. The example below shows the custom build command and how to configure watched files.
+
+**Example**: Tilt
+
+1. Clone the official Tilt examples: `git clone https://github.com/tilt-dev/tilt-example-java`
+2. `cd tilt-example-java/0-base`
+3. Modify your `Tiltfile` to look like this:
+
+    {{< code/copyable >}}
+    custom_build(
+        'example-java-image',
+        'pack build --pull-policy=if-not-present -e BP_LIVE_RELOAD_ENABLED=true example-java-image:dev',
+        ['pom.xml', 'bin/main'],
+        live_update = [
+            sync('./bin/main', '/workspace/BOOT-INF/classes'),
+        ],
+        tag="dev"
+
+    )
+    k8s_yaml('kubernetes.yaml')
+    k8s_resource('example-java', port_forwards=8000)
+    {{< /code/copyable >}}
+
+4. Execute `tilt up`.
+
+5. You should now be able to view your application running on your Kubernetes cluster by going to `http://localhost:8000` in your browser. In addition, if you edit files in your IDE, save and rebuild you should see those change live-reloaded into the application.
```

Comment:
> if you edit files in your IDE, save and rebuild you should see those change live-reloaded

This implies to the user that 'rebuild' is a separate step that they need to do. When live-reloading, simply saving a watched file triggers the rebuild. Unless the setup specifically has watching disabled while live reload is active, which I've personally never come across, I'd rewrite this as:

> when you save any files you've edited in your IDE, a rebuild occurs and you should see those changes live-reloaded